README.md

```diff
@@ -56,22 +56,22 @@ partial interface Navigator {
 };
 ```
 
-  * Sequences are returned for the properties as multiple may be available in the user's address book. User agents are encouraged to enable users to limit their selection.
-  * Support for additional properties can be added iteratively. Whether the returned data can (and should) be sanitized is a question that's unique to each property. At least one property must be provided.
+  * Sequences are returned for the properties as multiple values may be available in the user's address book. User agents are encouraged to enable users to limit their selection.
+  * Support for additional properties can be added iteratively. Whether the returned data can (and should) be sanitized is a question that's unique to each property. The user agent must provide at least one property.
   * Some future might include the ability to add contacts, or even _contact management_, so having an intermediary object on `navigator` helps extensibility.
```

Comment:
It's unclear who has to provide the one property. The sentence was also in passive voice.